<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreqFilter</class>
 <widget class="QWidget" name="FreqFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreqFilter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,1">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>原图的频谱图</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="srcLabel">
            <property name="minimumSize">
             <size>
              <width>428</width>
              <height>317</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>生成图的频谱图</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="dstLabel">
            <property name="minimumSize">
             <size>
              <width>428</width>
              <height>317</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>滤波模板的频谱图</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="filterLabel">
            <property name="minimumSize">
             <size>
              <width>428</width>
              <height>317</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9" stretch="2,0,1">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>高斯滤波</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>理想滤波</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>巴特沃斯滤波</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QRadioButton" name="lowRbtn">
                 <property name="text">
                  <string>低通</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">highOrlow</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="highRbtn">
                 <property name="text">
                  <string>高通</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">highOrlow</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>参数</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1">
            <item>
             <widget class="QGroupBox" name="rGroupBox">
              <property name="title">
               <string>通带半径</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QSpinBox" name="rSpinBox"/>
                 </item>
                 <item>
                  <widget class="QSlider" name="rSlider">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="btwGroupBox">
              <property name="title">
               <string>巴特沃斯次数</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QSpinBox" name="btwSpinBox"/>
                 </item>
                 <item>
                  <widget class="QSlider" name="btwSlider">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="sureBtn">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="closeBtn">
             <property name="text">
              <string>关闭</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="highOrlow"/>
 </buttongroups>
</ui>
